L1Trigger/TrackFindingTracklet/src/VMRouterTable.cc

```diff
@@ -98,6 +98,31 @@ void VMRouterTable::init(unsigned int layerdisk) {
       }
     }
   }
+
+  if (settings_.writeTable()) {
```

Comment:
in the spirit of adhering to CMSSW code rules, there are many "magic numbers" here. can they be constants? 

cf: https://github.com/cms-L1TK/cmssw/blob/L1TK-dev-11_1_0_pre8/L1Trigger/TrackFindingTracklet/interface/Settings.h#L19-L22
https://github.com/cms-L1TK/cmssw/blob/L1TK-dev-11_1_0_pre8/L1Trigger/TrackFindingTracklet/src/VMRouterTable.cc#L53
etc.

Reply:
Do you mean the constants that are defined inside the trklet namespace but outside the Settings class? There are a couple of those and they can probably be moved to be members of the class instead. If that's what you mean I'll do that and push a new commit. Or do you mean all the numbers defined as constants in the Settings.h file? Because there are more than 100 of those, and I would have thought the Settings class is the correct place to define them. Which do you mean?